notebooks/notebooks/paper/estimate-flops-adv.ipynb

```diff
@@ -273,46 +273,34 @@
      "name": "stdout",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    print_stats(df_cyclic.query("adv_steps &lt;= 545065216"))
</pre><p>Nit: We can do <code>adv_steps &lt; 545065216</code> since we only use games from adversaries with step count below 545065216 in order to train the adversary with step count 545065216?</p><p>Same thing with the pass adversary </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Good catch!

Reply:
This actually non-trivially reduces the compute needed for our strongest adversary.